app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -99,25 +99,25 @@ The salary you must be paid to get a visa varies according to  whether you are e
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £34,502
-* outer London, you must earn £32,407
-* London fringe, you must earn £29,344
-* the rest of England, you must earn £28,000
+* inner London, you must earn £36,745
+* outer London, you must earn £34,514
+* London fringe, you must earn £31,350
```

Comment:
@laurahermionetennant92 what's the difference between outer London and London fringe?

Reply:
No idea haha. But schools will know which category they fall into, which is why we suggest teachers check with them.